### PR TITLE
Remove docs example using internal pyregion._region_filter

### LINKED ISSUE #129: Private functions shouldn't be promoted to be used
While I'm hunting for some weird CI issues in astroquery, I've run into this in pyregion. IMO we shouldn'y promote usage of private functions in the `getting started` page, if this needs to be in the API then expose it properly in the API docs, etc.

` >>> import pyregion._region_filter as filter `

### PR DESCRIPTION
**Pull request by cdeil**
Fixes #129 .

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -213,6 +204,6 @@ necessary).
    :include-source:
 
 Note that this will fail if your template image is not a simple 2D image.
-To work around this you may use the ``shape`` optional argument of ``get_mask``: ::
+To work around this you may use the ``shape`` optional argument of `ShapeList.get_mask <pyregion.ShapeList.get_mask>`: ::
```
Comment (bsipocz) [inline docs/getting_started.rst]: this way of linking looks unusual, but I see why you did that.

**Comment by cdeil** [comment]
I'll just merge this now. @leejjoon or anyone - of course comments or follow-up PRs to improve the docs are welcome any time.

**Comment by cdeil** [comment]
Oh boy: https://travis-ci.org/astropy/pyregion/jobs/373880714#L838

@bsipocz - Thoughts? How about just removing the build against that old Numpy version?

**Comment by cdeil** [comment]
I removed the old Numpy 1.8 build from `.travis.yml` via fc3b979 in `master`.

**Pull request merged**